src/sql/magic.py

```diff
@@ -678,8 +678,11 @@ def _persist_dataframe(
 
 def set_configs(ip, file_path):
     """Set user defined SqlMagic configuration settings"""
+    is_notebook = ip.__class__.__name__ == "ZMQInteractiveShell"
     sql = ip.find_cell_magic("sql").__self__
-    user_configs = util.get_user_configs(file_path, ["tool", "jupysql", "SqlMagic"])
+    user_configs = util.get_user_configs(
+        file_path, ["tool", "jupysql", "SqlMagic"], is_notebook
+    )
```

Comment:
we don't need this `is_notebook` logic, the display module already takes care of displaying an HTML version of the message if running on a notebook

Reply:
I had to add it because it prints `<IPython.core.display.HTML>` when running in terminal and also in tests. So I wasn't able to test the actual output message @edublancas

Reply:
the display methods print plain text when they're executed in the terminal, check our existing tests:

```
In [1]: from sql import display

In [2]: display.message("hello")
hello

In [3]: display.message_success("hello")
hello

In [4]: display.message_warning("hello")
hello

In [5]:

```

Reply:
I'm not sure what I'm missing here. I did try both on notebook and terminal. I have used `message_html` and not `message` because `message` doesn't display a clickable link.

<img width="1249" alt="Screenshot 2023-08-29 at 12 23 59 PM" src="https://github.com/ploomber/jupysql/assets/29096135/953c5dfc-0f6f-42c1-9f0f-a5da4b157321">

The same on terminal (HTML version is displaying the class name only).

<img width="893" alt="Screenshot 2023-08-29 at 12 25 14 PM" src="https://github.com/ploomber/jupysql/assets/29096135/4d1d7f97-220c-4446-aef5-dcd2aca0be40">

 I found only one existing function which uses `message_html` and in the testcase I see a similar ouptut:

<img width="774" alt="Screenshot 2023-08-29 at 11 54 34 AM" src="https://github.com/ploomber/jupysql/assets/29096135/77139e22-7058-4694-938c-11a9b1ed59aa">

Are you suggesting I replace `message_html` with `message`? Or should I modify the `Message` class to convert html links in plain text to anchor tags?
 @edublancas

Reply:
I understand what's the problem now. Use message_html, and in your tests, you can mock message_html to test whether it was called with the right argument or not.

I'll open an issue so we tackle this problem, since there are limitations with the current display.py module

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/823/commits/90f6c0666ff7f87764ecd3e37c73f96d10fcbea5)